cmd/juju/cloud/show.go

```diff
@@ -130,6 +142,50 @@ func makeCloudDetails(cloud jujucloud.Cloud) *cloudDetails {
 		result.Regions = append(result.Regions, yaml.MapItem{r.Name, r})
 		result.RegionsMap[region.Name] = r
 	}
+	modelConfig := make(map[string]interface{})
+	for attr, value := range result.BootstrapConfig {
+		if !bootstrap.IsBootstrapAttribute(attr) {
+			modelConfig[attr] = value
+			delete(result.BootstrapConfig, attr)
+		}
+	}
+	if len(modelConfig) > 0 {
+		result.ModelConfig = modelConfig
+	}
+	// Look for config options specific to the cloud provider, if available.
+	provider, err := environs.Provider(cloud.Type)
+	if err != nil {
+		return result
+	}
+	// Not all providers have implemented the ProviderSchema interface,
+	// e.g. joyent, double check before using.
+	if m, ok := interface{}(provider).(environs.ProviderSchema); ok {
```

Comment:
Can we invert this logic to return early if !ok
Also, we don't need the "interface{}"...
m, ok := provider.(environs.ProviderSchema)